test/linter.js

```diff
@@ -7,15 +7,15 @@ var _ = require('lodash'),
   utils = require('./util/utils');
 
 describe('AutoRest Linter validation:', function () {
-  let swaggersToProcess = utils.getFilesChangedInPR();
+  let configsToProcess = utils.getConfigFilesChangedInPR();
   // Useful when debugging a test for a particular swagger. 
   // Just update the regex. That will return an array of filtered items.
-  // swaggersToProcess = swaggersToProcess.filter(function(item) {
+  // configsToProcess = configsToProcess.filter(function(item) {
   //   return (item.match(/.Microsoft.ContainerRegistry.*2017-03-01.*/ig) !== null);
   // });
-  _(swaggersToProcess).each(function (swagger) {
-    it(swagger + ' should honor linter validation rules.', function (done) {
-      var cmd = 'autorest --azure-validator=true --input-file=' + swagger + ' --message-format=json';
+  _(configsToProcess).each(function (config) {
+    it(config + ' should honor linter validation rules.', function (done) {
+      var cmd = `autorest --azure-validator ${config} --message-format=json`;
       console.log(`Executing: ${cmd}`);
```

Comment:
all (autorest enabled) tools should go this route sooner or later... i.e. call model/semantic validator through AutoRest (benefit of uniform external file resolution, suppressions, uniform output format, ...)